pkgs/development/python-modules/disnake/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, fetchFromGitHub
+, libopus
+, pynacl
+, pythonOlder
+, withVoice ? true
+, ffmpeg
+}:
+
+buildPythonPackage rec {
+  pname = "disnake";
+  version = "2.9.2";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "DisnakeDev";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-J7nNVHhkQ9ekGjioObxaTL0OhjLjHHYlgbPdAG9Vz4o=";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+  ] ++ lib.optionals withVoice [
+    libopus
+    pynacl
+    ffmpeg
+  ];
+
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```
Please don't override the patchPhase. This makes it difficult to patch.